src/resources/js/components/Presentation.vue

```diff
@@ -45,17 +45,16 @@ export default {
       polls: [],
     };
   },
+  watch: {
+    /* For search presentation issue, watcher will look for route alteration
+     * and launch code if there is. (eg. if we push the same route with other params)
+     */
+    $route() {
+      this.getPresentation(this.$route.params.idPresentation);
+    },
+  },
   beforeMount() {
-    alert = this.parentRefs.alert;
```

Comment:
Watch out you removed the alert component declaration

Reply:
ok corrected on [5e41da](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/152/commits/5e41da84ab6163ab4f780d0e817fabd635deb866)